src/platform/monitoring/DowntimeNotification/util/helpers.js

```diff
@@ -104,19 +107,36 @@ export function getSoonestDowntime(serviceMap, serviceNames) {
 }
 
 /**
- * Determines whether there is a global downtime in progress
- * @param {object} downtimeWindow Optional arg to set arbitrary downtime
- * @returns {boolean} true if the current time is within the downtime window,
- *     false if not
+ * Retrieves a list of global downtimes from a cached JSON file and gets the
+ * downtime that includes the current time.
+ * @returns {object} A global downtime window that covers the current time
+ *     if it exists and null if not
  */
-export const isGlobalDowntimeInProgress = (
-  downtimeWindow = scheduledDowntimeWindow,
-) => {
-  const { downtimeStart, downtimeEnd } = downtimeWindow;
-
-  return (
-    !environment.isLocalhost() &&
-    moment().isAfter(downtimeStart) &&
-    moment().isBefore(downtimeEnd)
-  );
-};
+export const getCurrentGlobalDowntime = (() => {
```

Comment:
@ncksllvn @kelsonic Just a heads up that we've set up a way to configure and retrieve global downtime windows without hard-coding. The way this works is that we can set maintenance windows in PagerDuty for the "Internal: API" service, which is represented by the `global` service name, and those maintenance windows will be stored in a file in S3 as JSON.